client/ayon_core/hosts/houdini/api/pipeline.py

```diff
@@ -262,7 +262,29 @@ def parse_container(container):
         dict: The container schema data for this container node.
 
     """
-    data = lib.read(container)
+    # Read only relevant parms
+    # TODO: Clean up this hack replacing `lib.read(container)`
+    import six
+    import json
+    from .lib import JSON_PREFIX
```

Comment:
Why are imports inside the function?

Reply:
Just me trying to be quick when I implemented this 'fix' at the time.